### PR TITLE
Upgrade to Kinto 1.3.1.

### PR DESCRIPTION
**Pull request by n1k0**

### CODE REVIEW

**Comment by n1k0** [comment]
Don't merge this, as there's a [problem with the Kinto test server installation process](https://github.com/mozilla-services/kinto/issues/142). Also, it seems that our builds are marked green even if failures occured (#74).

**Comment by n1k0** [comment]
Refs #76 — Now we have failing builds we're properly notified about, which we can sanely start working on fixing.

**Comment by n1k0** [comment]
```
Command "/usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-build-utaanN/venusian/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-z8EqQ4-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-utaanN/venusian
```

Le sigh. Have to fallback to use sudo.

**Comment by n1k0** [comment]
```
Traceback (most recent call last):
  File "/usr/local/bin/pserve", line 7, in <module>
    from pyramid.scripts.pserve import main
  File "/usr/local/lib/python2.7/dist-packages/pyramid/scripts/pserve.py", line 33, in <module>
    from pyramid.paster import setup_logging
  File "/usr/local/lib/python2.7/dist-packages/pyramid/paster.py", line 10, in <module>
    from pyramid.scripting import prepare
  File "/usr/local/lib/python2.7/dist-packages/pyramid/scripting.py", line 1, in <module>
    from pyramid.config import global_registries
  File "/usr/local/lib/python2.7/dist-packages/pyramid/config/__init__.py", line 20, in <module>
    from pyramid.authorization import ACLAuthorizationPolicy
  File "/usr/local/lib/python2.7/dist-packages/pyramid/authorization.py", line 9, in <module>
    from pyramid.security import (
  File "/usr/local/lib/python2.7/dist-packages/pyramid/security.py", line 13, in <module>
    from pyramid.threadlocal import get_current_registry
  File "/usr/local/lib/python2.7/dist-packages/pyramid/threadlocal.py", line 3, in <module>
    from pyramid.registry import global_registry
  File "/usr/local/lib/python2.7/dist-packages/pyramid/registry.py", line 5, in <module>
    from zope.interface.registry import Components
```

Le sigh++.

**Comment by Natim** [comment]
If you use a virtualenv you don't need sudo.

**Comment by n1k0** [comment]
> If you use a virtualenv you don't need sudo.

You will laugh at this, I'm sure; you need sudo to install virtualenv.

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,9 @@ language: node_js
...
 - '0.12'
-services:
-- redis-server
 before_install:
 - sudo pip install virtualenv
-- test/scripts/setup-kinto.sh
+- virtualenv `pwd`/.env
+- source .env/bin/activate
+- pip install kinto==1.3.1
+- pserve test/kinto.ini &
```
Comment (Natim) [inline .travis.yml]: You can use directly ``.env/bin/pserve``

File: .travis.yml [hunk 2]
```diff
@@ -2,8 +2,9 @@ language: node_js
...
 node_js:
 - '0.12'
-services:
-- redis-server
 before_install:
 - sudo pip install virtualenv
-- test/scripts/setup-kinto.sh
+- virtualenv `pwd`/.env
+- source .env/bin/activate
+- pip install kinto==1.3.1
```
Comment (Natim) [inline .travis.yml]: You can use directly ``.env/bin/pip``

File: .travis.yml [hunk 3]
```diff
@@ -2,8 +2,9 @@ language: node_js
 sudo: true
 node_js:
 - '0.12'
-services:
-- redis-server
 before_install:
 - sudo pip install virtualenv
-- test/scripts/setup-kinto.sh
+- virtualenv `pwd`/.env
```
Comment (Natim) [inline .travis.yml]: You should be able not to use `pwd`

File: .travis.yml [hunk 4]
```diff
@@ -2,8 +2,9 @@ language: node_js
...
+- source .env/bin/activate
```
Comment (Natim) [inline .travis.yml]: You can get rid of this if you use directly ``.env/bin/*``

**Comment by n1k0** [comment]
>  This job ran on our legacy infrastructure.

Oh well. I wish we could avoid using sudo, though our dependency to python stuff seems to prevent this quite heavily. Feel free to spawn a new issue if it eventually becomes a show-stopper. 

I'll rebase and land this as we badly need to move forward.

**Pull request merged**